package.json

```diff
@@ -31,11 +31,11 @@
     "prettier": "prettier --single-quote --print-width 120 --write \"{blueprints,config,lib,app,addon,addon-test-support,tests,tests-node}/**/*.js\""
   },
   "dependencies": {
+    "@ember-intl/broccoli-cldr-data": "^3.0.0",
```

Comment:
Yeah this is a breaking change (as you mentioned).

We'll need to publish a new major on ember-intl.